core/src/components/picker-column/picker-column.scss

```diff
@@ -22,17 +57,46 @@
 
   // Hide scrollbars on Firefox
   scrollbar-width: none;
-
-  text-align: center;
 }
 
 /**
  * Hide scrollbars on Chrome and Safari
  */
-:host::-webkit-scrollbar {
+.picker-opts::-webkit-scrollbar {
   display: none;
 }
 
+::slotted([slot="prefix"]), ::slotted([slot="suffix"]) {
+  display: flex;
+
+  flex: 1;
+  align-items: center;
+}
+
+::slotted([slot="prefix"]) {
```

Comment:
We might want to have text truncate with ellipses otherwise it will spill outside of the highlight area: 
![image](https://github.com/ionic-team/ionic-framework/assets/2721089/74af817f-d533-4763-a91e-d0627344eca1)


Prefix/suffix text should generally be short anyways.

Reply:
Done in https://github.com/ionic-team/ionic-framework/pull/28603/commits/4547e18428b47c13eb66ecd94884a4e7c0d79638.

Reply:
Added in https://github.com/ionic-team/ionic-framework/pull/28603/commits/15dd7a8df15d2cae5d7d86dc4a7de97285410944, updating screenshots now 👍